test/e2e/framework/skipper/skipper.go

```diff
@@ -56,8 +56,18 @@ type SkipPanic struct {
 	FullStackTrace string // A full stack trace starting at the source of the failure
 }
 
+const GINKGO_PANIC = `
+Your test failed.
```

Comment:
"Your test failed." -> "Your test was skipped."

If it wasn't for this slight difference, skipper should reference the same string from ginkgowrapper.

Also, don't export this string. It's an implementation detail.